packages/core/config/jest.setup.web.js

```diff
@@ -2,11 +2,24 @@ require('@testing-library/jest-dom')
 require('whatwg-fetch')
 
 const { server } = require('@redwoodjs/testing')
+const camelCase = require('lodash/camelCase')
+const { PrismaClient } = require('@prisma/client')
```

Comment:
I would like to import the `db.js` from the API side of the app, but Jest won't transpile it because it's outside of the rooDir (I think).

Reply:
I can take a look at this, which steps should I take to reproduce this?

Reply:
1. Checkout `https://github.com/RobertBroersma/redwood-testing`
2. Checkout this PR
3. Replace in this file the direct `PrismaClient` import with the `db.js` import.
4. Run `yarn rw test web` in the `redwood-testing` repo

Thanks!